egs2/librispeech_100/asr_transducer1/run.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -o pipefail
+
+train_set="train_clean_100"
+valid_set="dev"
+test_sets="test_clean test_other dev_clean dev_other"
+
+asr_config=conf/train_conformer-rnn_transducer.yaml
+inference_config=conf/decode.yaml
+inference_model=valid.loss.ave_10best.pth
+
+./asr.sh \
+    --asr_task asr_transducer \
+    --skip_data_prep false \
+    --skip_train false \
+    --skip_eval false \
```

Comment:
Do you need these lines?

Reply:
Hi Shinji,

Sorry about the delay, This recipe is not up-to-date as I refactored some parts of the standalone implementation. I'll make a PR for the changes and update this one!

(And no, I don't think we need these lines! I think it's a remaining of some exp)